benefits/oauth/views.py

```diff
@@ -66,10 +66,12 @@ def authorize(request):
 
     if verifier_claim:
         userinfo = token.get("userinfo")
-        # the claim comes back in userinfo like { "claim": "True" | "False" }
-        claim_flag = (userinfo.get(verifier_claim) if userinfo else "false").lower() == "true"
-        # if userinfo contains our claim and the flag is true, store the *claim*
-        stored_claim = verifier_claim if claim_flag else None
+
+        if userinfo and userinfo.get(verifier_claim) is not None:
```

Comment:
Maybe `userinfo.get(verifier_claim)` once and store in a variable along with `userinfo`.

Otherwise this is much cleaner 👍